Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.3
+
+RUN touch ~/.gemrc && echo "gem: --no-ri --no-rdoc" >> ~/.gemrc
+
+WORKDIR /app/
+ADD . /app/
+RUN bundle install
```

Comment:
nit: we only need it if we intend to run this image not as part of `docker-compose`